dlite_entities_service/models/dlite.py

```diff
@@ -0,0 +1,111 @@
+"""DLite model."""
+from __future__ import annotations
+
+from typing import Annotated
+
+from pydantic import AliasChoices, Field, field_validator
+from pydantic.networks import AnyHttpUrl
+
+from dlite_entities_service.models.soft5 import SOFT5Entity, SOFT5Property
+from dlite_entities_service.models.soft7 import SOFT7Entity, SOFT7Property
+
+
```

Comment:
Would it make sense to perhaps have a` BaseDLiteProperty`& `BaseDLiteEntity `classes, then use multiple inheritance, e.g.` class DLiteSOFT5Property(BaseDLiteProperty, SOFT5Property) ` ` class DLiteSOFT7Property(BaseDLiteProperty, SOFT7Property)` . This will cut down a bit on the redundant code. [I do remember you talking about this a bit this morning so maybe there is a good reason to keep them separate.

Reply:
I did think about this, and it would be a bit nicer. I thought about doing it this way as well, but I am unsure whether pydantic will properly propagate and merge the two models. So I recognized I would need to test a bit more if I did it this way. Hence, I skipped it in the first draft here. But I can look into it.

Reply:
done